src/ACL.cpp

```diff
@@ -155,13 +155,17 @@ QFlags<ChanACL::Perm> ChanACL::effectivePermissions(ServerUser *p, Channel *chan
 						granted |= Kick;
 					if (acl->pAllow & Ban)
 						granted |= Ban;
+                    if (acl->pAllow & ResetComment)
+                        granted |= ResetComment;
+                    if (acl->pAllow & ResetTexture)
+                        granted |= ResetTexture;
```

Comment:
Indentation doesn't match the rest of the code. Spaces vs tabs? (also applies to other places in your changes)